### PR TITLE
style(aio): updated padding-right to h3 in _heading-anchors.scss

### LINKED ISSUE #22407: Docs - h3 heading misaligned inside div
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior
The `h3` heading for "[TypeScript interfaces aren't valid tokens](https://angular.io/guide/dependency-injection#typescript-interfaces-arent-valid-tokens)" is misaligned within its `div`.

## Expected behavior
The `h3` heading should be visually left aligned within it's `div` to maintain consistency with other sections.

## Minimal reproduction of the problem with instructions
Open Angular's documentation for  "[TypeScript interfaces aren't valid tokens](https://angular.io/guide/dependency-injection#typescript-interfaces-arent-valid-tokens)"

## What is the motivation / use case for changing the behavior?
To maintain consistency with other sections in the Angular documentation.

## Environment

Angular version: 1.6.9

Browser:
- [x] Chrome (desktop) version 64.0.3282.167 (Official Build) (64-bit)
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [x] Firefox 58.0.2 (64-bit)
- [x] Safari (desktop) Version 11.0.3 (13604.5.6)
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
## GIF of Bug

angularheadingbug

Upon closer inspection inside Chrome's dev tools, it looks like the heading isn't being properly aligned for a CSS reason:

screen shot 2018-02-23 at 12 34 03 pm

## Potential Fix
A potential fix might be adding `padding-left: 34px;` to `.l-sub-section h3` found in `_subsection.scss:13`. This would result in the following:

screen shot 2018-02-23 at 12 37 53 pm

### PR DESCRIPTION
**Pull request by haoRchen**
The h3 element is overflowing over its surrounding div element. Modified padding-left to align consistently with the remainder of div contents.

fixes: #22407

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

issue

Issue Number: 22407

## What is the new behavior?
fixed

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by haoRchen** [comment]
In addition, I noticed that the header is not being aligned properly in mobile/responsive views. See below:

mobile view error

Please advise.

File: aio/src/styles/2-modules/_subsection.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
     h3 {
         margin: 8px 0 0;
+        padding-left: 34px;
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_subsection.scss]: As mentioned by @petebacondarwin in https://github.com/angular/angular/issues/22407#issuecomment-368243696, the styles for `<h3>` are fine.

You should instead change the `.heading-anchor` styles in [_heading-anchors.scss][1]. More specifically, I think you should change [this line][2]:

```diff
padding-right: 24px;
padding-right: 64px;
```

(For reference, this was accidentally broken in e3e7044d0.)

[1]: https://github.com/haoRchen/angular/blob/5ee251ea0f1fde91b163eb2df08abf8b9d761236/aio/src/styles/2-modules/_heading-anchors.scss
[2]: https://github.com/haoRchen/angular/blob/5ee251ea0f1fde91b163eb2df08abf8b9d761236/aio/src/styles/2-modules/_heading-anchors.scss#L34

**Review by gkalpak — CHANGES_REQUESTED** [review]

**Comment by haoRchen** [comment]
I've applied your fix @gkalpak, please let me know if theres anything else I can do. See below for some updated screenshots: 
fix
side fix

**Comment by SeanPrashad** [comment]
After looking further into other documentation with @haoRchen, we stumbled across another section in his [patch mentioned above](https://pr22431-8a429a6.ngbuilds.io/) that has a formatting issue [here](https://pr22431-8a429a6.ngbuilds.io/guide/dependency-injection-in-action#take-a-break):

post-dev-suggestion-fix

**Comment by haoRchen** [comment]
https://github.com/angular/angular/pull/22431/commits/2dbbeaad3b4617ffcb6e6e3c18132a63a54d7c95 addresses [@SeanPrashad 's comment](https://github.com/angular/angular/pull/22431#issuecomment-368376667). 

After fix screenshot:
fix for take a break

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**